### PR TITLE
Support versions

### PR DESCRIPTION
**Pull request by myakove**
Support cluster version from user input based on stream (AWS)/channel-group (ROSA).
Destroy clusters if create fail.

### CODE REVIEW

File: openshift_cli_installer/utils/cluster_versions.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+        click.echo(f"Cluster version set to {cluster_data['version']}")
+
+    return clusters
+
+
+def filter_versions(version, base_versions_dict, platform, stream):
+    versions_dict = {}
+    version_key = get_split_version(version=version)
+    x86_64_str = "-x86_64"
```
Comment (rnetser) [inline openshift_cli_installer/utils/cluster_versions.py:72]: make both `x86_64` and `-x86_64` constants

**Review by rnetser — CHANGES_REQUESTED** [review]

File: openshift_cli_installer/tests/all_rosa_versions.py [hunk 2]
```diff
@@ -0,0 +1,319 @@
+BASE_AVAILABLE_VERSIONS_DICT = {
```
Comment (omrirh) [inline openshift_cli_installer/tests/all_rosa_versions.py:1]: is there a dynamic way to fetch all rosa versions?

**Review by rnetser — DISMISSED** [review]

File: openshift_cli_installer/tests/all_rosa_versions.py (hunk 2 above)
Comment (myakove) [reply openshift_cli_installer/tests/all_rosa_versions.py:1]: This is for the tests, why would I want this to be dynamic?

Comment (omrirh) [reply openshift_cli_installer/tests/all_rosa_versions.py:1]: so that we won't need to update new releases manually

**Review by omrirh — APPROVED** [review]

**Review by rnetser — APPROVED** [review]

**Pull request merged**